scripts/ci-e2e.sh

```diff
@@ -59,6 +59,9 @@ if [ -n "${BOSKOS_HOST:-}" ]; then
     echo "error getting account from boskos" 1>&2
     exit "${checkout_account_status}"
   fi
+
+  python -u hack/heartbeat_account.py >> $ARTIFACTS/boskos.log 2>&1 &
```

Comment:
i had made a boo boo here as well. we need `$ARTIFACTS/boskos.log` to be `$ARTIFACTS/logs/boskos.log` and we need to ensure that the directory is created

Reply:
I've been dumping all of the other logs directly under artifacts for now, so this is probably fine for now